src/proto/nc_redis.c

```diff
@@ -971,6 +974,11 @@ redis_parse_req(struct msg *r)
                     break;
                 }
 
+                if (str6icmp(m, 'o', 'b', 'j', 'e', 'c', 't')) {
```

Comment:
Just this change isn't enough to support `OBJECT` - twemproxy will treat the first string after the command name as the key (to shard on), but here, the subcommand is in that position, so it will go to the wrong server when there's more than one server. https://redis.io/commands/object

Reply:
the main purpose for supporting OBJECT is to get the **encoding** sub-command. even on sharding scenario, why we can't assume that all object have same encoding?

client app must understand that the answer for other sub-command other than **encoding** (refcount, freq, idletime) may differ from one server to another. 
even for that. why we can't support this OBJECT command?